### ISSUE TITLE
translate and loops

### ISSUE THREAD

**Issue by buraktamturk** [Anthony-Gaudino/jekyll-multiple-languages-plugin#66]
Hello,

I can't use the translate tag with for loops. Is it supported?

my en.yml file

```yaml
symbols:
  - title: First
    message: Message
  - title: Second
    message: Message
```

```
{% for a in translate symbols %}
        <li>
            <h2>{{ a.title }}</h2>
            <p>{{ a.message }}</p>
        </li>
{% endfor %}
```

**Comment by Anthony-Gaudino** [comment]
Hi @buraktamturk, sorry for taking so long to answer you.

I released version 1.4.2 with a small patch that allows you to access the `site.translations` hash. This way you can use it in a Liquid for loop like this:

```liquid
{% for item in site.translations[site.lang]["my_nested_yaml_collection"] %}
    <p>{{ item[0] }} -> {{ item[1] }}</p>
{% endfor %}
```

----

For the following YAML:

```yaml
symbols:
  -
    title: First
    message: Message
  -
    title: Second
    message: Message
```

The following code should work:

```liquid
{% for item in site.translations[site.lang]["symbols"] %}
   <li>
      <h2>{{ item["title"] }}</h2>
      <p>{{ item["message" }}</p>
   </li>
{% endfor %}
```

If you find any trouble please feel free to comment bellow.

**Comment by buraktamturk** [comment]
Hello, thanks for the work! I think it'll fit perfectly to my use case, and I'll test whenever possible soon.
